data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,399 @@
+# Copyright (c) 2023 by RisingLeaf (https://github.com/RisingLeaf)
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Drak guarding Incipias Space"
```

Comment:
```suggestion
mission "Drak guarding Incipias space"
```
Capitalization is different in the below condition/this is how the other drak missions are capitalized. Also, side thought - this mission should probably be in the drak missions file; the other ones are (Remnant, Korath, etc.).

Reply:
moved it